packages/terra-show-hide/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "terra-show-hide",
+  "main": "lib/ShowHide.js",
+  "version": "0.0.0",
+  "description": "Show/Hide component that previews a small section of content or shows the full content with the click of a button",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-core.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-show-hide",
+    "ShowHide",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-core/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-core#readme",
+  "peerDependencies": {
+    "react": "^16.2.0",
+    "react-dom": "^16.2.0",
+    "terra-base": "^3.1.0"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "prop-types": "^15.5.8",
+    "terra-doc-template": "^1.2.0",
+    "terra-icon": "^2.19.0",
+    "react-intl": "^2.4.0",
```

Comment:
What are `react-intl` & `react-test-renderer` included as dependencies?

Reply:
react-intl definitely should not be